### PR TITLE
[KYUUBI #3070][DOC] Add a doc of the Hudi connector for the Flink SQL Engine

### PR DESCRIPTION
**Pull request by deadwind4**
### _Why are the changes needed?_

Add a doc of the Hudi connector for the Flink SQL Engine

### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [ ] [Run test](https://kyuubi.apache.org/docs/latest/develop_tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

**Comment by yaooqinn** [comment]
any updates @deadwind4

File: docs/connector/flink/hudi.rst [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+
+Hudi Integration
+----------------
+
+To enable the integration of kyuubi flink sql engine and Hudi through
+Catalog APIs, you need to:
+
+- Referencing the Hudi :ref:`dependencies`
+
+.. _dependencies:
```
Comment (pan3793) [inline docs/connector/flink/hudi.rst:38]: let rename it to avoid conflict, see https://github.com/apache/incubator-kyuubi/pull/3161

Comment (deadwind4) [reply docs/connector/flink/hudi.rst]: done

**Review by pan3793 — APPROVED** [review]

**Comment by pan3793** [comment]
Thanks, merging to master

**Pull request closed without merging**